deploy-service/common/src/main/resources/sql/deploy.sql

```diff
@@ -36,6 +36,7 @@ CREATE TABLE IF NOT EXISTS environs (
     state               VARCHAR(32)   NOT NULL,
     cluster_name        VARCHAR(128),
     max_parallel_rp     INT           NOT NULL DEFAULT 1,
+    override_policy     VARCHAR(32)   NOT NULL,
```

Comment:
can we create a update_plan_2016xxxxx.txt just like the existing ones ( so we can follow it to upgrade production DBs). I assume in the plan we will add the column with the default "OVERRIDE" option. Otherwise, looks good to me. Thanks!